MyHashMap.java

```diff
@@ -0,0 +1,85 @@
+class MyHashMap {
+
+    // Time Complexity : 0(n)
+// Space Complexity : 0(n)
```

Comment:
Please try to implement double hashing technique as well.

Reply:
Thank you for the review. Will do the same shortly.